### DISCUSSION TITLE
Emulator cannot working.

### DISCUSSION THREAD

**Discussion by GabrielBRDeveloper** [Genymobile/feedback#53] [Genymotion Desktop] [answered]
The emulator freeze on device boot screen, and cannot respond more. 
In addition, after closing and opening genymotion, the device continues running in the background, being possible to turn it off only on the virtualbox screen.

I tried: 
   - Restart PC
   - Reinstall vbox.
   - Reinstall Emulator.

My PC:
  - OS: Linux Mint 21.1 (Ubuntu 22.04 - jammy)
  - CPU: Xeon E5-2620 v3
  - Hypervisor: Virtualbox
  - Genymotion version: 3.3.3
  - Vbox version: 7.0.8 (lasted)
  - Package font: .bin from offical website.

Freezed screen:
![image](https://github.com/Genymobile/feedback/assets/97042217/f8da7571-6ff7-4296-8d42-a124f2ac157d)

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @GabrielBRDeveloper,
We are terribly sorry but this board is not suitable for technical assistance. Please visit our help center to contact our support team at https://support.genymotion.com/hc/en-us. Thank you!
